src/components/Text/Text.module.css

```diff
@@ -5,8 +5,78 @@
 \*------------------------------------*/
 
 /**
- * 1) 
+ * 1) h1, h2, h3, h4, h5, h6
+ * 2) Styles are either controlled by as or size declaration
  */
-/* stylelint-disable-next-line block-no-empty */
-.text {
+.text,
+.text--body, 
+.text--md {
+  @mixin reset;
+  
+  @mixin eds-theme-typography-body-text;
+}
+
+/**
+ * Sizes
+ */
+.text--lg {
+  @mixin eds-theme-typography-body-text-lg;
+}
+.text--sm {
+  @mixin eds-theme-typography-body-text-sm;
+}
+.text--xs {
+  @mixin eds-theme-typography-body-text-xs;
+}
+.text--caption {
+  @mixin eds-theme-typography-caption-text-md;
+}
+.text--overline {
+  @mixin eds-theme-typography-kicker;
```

Comment:
For some reason, this isn't being capitalized https://61313967cde49b003ae2a860-hxziyqzwqa.chromatic.com/?path=/story/atoms-text-text--overline, please lmk what I can fix here

Reply:
The kicker uses preset `-008`, which isn't capitalized. you can see the full (WIP) typescale here: https://www.figma.com/file/IfMWAtWqNrQprImlco4pi0/EDS---UI-Foundations?node-id=62%3A3269

Would suggest updating `eds-theme-typography-kicker` to point to either preset `-011` or `-012` instead

Reply:
Was pointing to `-011` but there was a duplicate with `-008`, removed